e2e-tests/IIoTPlatform-E2E-Tests/Config/IcontainerRegistryConfig.cs

```diff
@@ -0,0 +1,38 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+
+namespace IIoTPlatform_E2E_Tests.Config {
+    /// <summary>
+    /// Container registry client configuration
+    /// </summary>
+    public interface IContainerRegistryConfig {
```

Comment:
Please fix filename, it should be IContainerRegistryConfig.cs

Reply:
fixed

Reply:
@dacolgit, the file is still called `IcontainerRegistryConfig.cs`

Reply:
+1 still need to be fixed

---
In reply to: [528568809](https://github.com/Azure/Industrial-IoT/pull/916#discussion_r528568809) [](ancestors = 528568809)

Reply:
It's really strange. I changed the name, but the change of the letter to capital is not seen as a change from git

Reply:
fixed now